src/index.less

```diff
@@ -161,6 +161,25 @@ ovh-chatbot { /* stylelint-disable-line selector-type-no-unknown */
       padding: 17px 17px 0 17px;
       height: ~'calc(100% - @{chatbot-header-height} - @{chatbot-footer-height})';
 
+      .chatbot-banner {
+        margin: -17px -17px 0 -17px;
+
+        .ovh-bandeau {
```

Comment:
Mix between EN/FR selector name.

Reply:
It's dydu... I'm not the one making this name.